source/uwp/Renderer/lib/Util.cpp

```diff
@@ -441,6 +447,10 @@ HRESULT GenerateContainedElementsProjection(
             RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveToggleInput>(
                 &projectedContainedElement, std::AdaptivePointerCast<AdaptiveSharedNamespace::ToggleInput>(containedElement)));
             break;
+        case CardElementType::ActionSet:
+            RETURN_IF_FAILED(MakeAndInitialize<::AdaptiveNamespace::AdaptiveActionSet>(
+                &projectedContainedElement, std::dynamic_pointer_cast<AdaptiveCards::ActionSet>(containedElement)));
```

Comment:
>dynamic_pointer_cast [](start = 49, length = 20)

Why is this `dynamic_pointer_cast` instead of `AdaptivePointerCast`?

Reply:
Likely because i wrote this a long time ago, before there was an AdaptivePointerCast :)

---
In reply to: [249154942](https://github.com/Microsoft/AdaptiveCards/pull/2244#discussion_r249154942) [](ancestors = 249154942)